freqtrade/configuration/arguments.py

```diff
@@ -41,7 +43,7 @@
 ARGS_PLOT_PROFIT = ["pairs", "timerange", "export", "exportfilename", "db_url",
                     "trade_source", "ticker_interval"]
 
-NO_CONF_REQURIED = ["download-data", "plot-dataframe", "plot-profit"]
+NO_CONF_REQURIED = ["download-data", "list-timeframes", "plot-dataframe", "plot-profit"]
```

Comment:
i suspect `"list-timeframes"` should now be moved to `NO_CONF_ALLOWED` ...

It doesn't make sense to use a "real" config for this command i think.

Reply:
Are you sure? Currently, NO_CONF_ALLOWED contains only commands that use args and do not utilize config...

For this command is convenient to read the default config and print timeframes for the exchange set in it. This is from one point of view. From other point of view, right, this command may require exchange be set with the `--exchange` or config be explicitely specified with `-c`. I don't know which option is best here...

Reply:
> Now the behavior is strange: the config.json exists, I want simply list the timeframes for another exchange (so I specified it with --exchange, because I know that my default config.json is set to work with other exchange), and it tells me something about non-existing pairs...

Well i was basing this on your comment above, that it'll be strange if you have config.json and want to list timeframes for another exchange ...

both are valid options and depend on what information you're looking for ...